docs/website/contents/explanation/consensus-protocol.md

```diff
@@ -0,0 +1,267 @@
+
+- 1. **Operational Certificate Issue Number**:
+If both chain tips are from the same issuer and slot, the one with the higher `opcert` issue number is preferred.
+This allows a stake pool operator to replace a compromised hot key with a new one and still have their blocks take precedence.
+
+- 2. **VRF Tiebreaker**:
+If the `opcert` check is inconclusive (which is common when two pools are elected in the same slot), the chain with the lower VRF value at its tip is preferred.
+This avoids always picking the first block to arrive, which could encourage centralization to reduce latency (see ["The Frankfurt Problem"](https://github.com/IntersectMBO/ouroboros-consensus/blob/40d77fdd74a9b2b2a1d112a0b836b5cb8026c88c/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L227)). The VRF value used for tiebreaking (non-range extended) is uncorrelated to the leader VRF value and typically results in a uniformly random decision. Depending on the `ChainOrderConfig` for Praos, there are two [flavors](https://github.com/intersectmbo/ouroboros-consensus/blob/010b374c54f4d2f485ab114f702db6ec3b7a8f95/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L75) that determine when this tie-breaker comparison takes place.
+    - `UnrestrictedVRFTiebreaker`: With this flavor, VRF tiebreakers are always compared. This has been the standard behavior for all eras before Conway.
+    - `RestrictedVRFTiebreaker`: This flavor restricts the VRF tiebreaker comparison to situations where the slot numbers of the chain tips differ by at most a specified maximum distance (`maxDist`).
+The primary motivation for this restriction is to favor blocks that were diffused earlier (in earlier slots) over those diffused later, even if the later block has a "better" VRF tiebreaker value. This aims to mitigate [issues](https://github.com/IntersectMBO/ouroboros-network/issues/2913) caused by poorly configured or resource-constrained pools that might diffuse blocks later.
+
+### On the Transitivity of Praos Chain Ordering
+
+Praos chain ordering is **not transitive**, regardless of the VRF tiebreaker flavor.
+
+Consider the following select views where chains `A`, `B`, and `C` have the same length:
+
+|                 | A | B | C |
+|-----------------|---|---|---|
+| Issuer          | x | y | x |
+| Slot            | 0 | i | 0 |
+| `opcert` number | 2 | j | 1 |
+| VRF             | 3 | 2 | 1 |
+
+Lower-case letters stand for arbitrary values (two letters designate the same value if and only if they are same letter).
+
+In this example we have:
+
+- `B` is preferred over `A`, since `B` has lower VRF than `A`.
+- `C` is preferred over `B`, since `C` has lower VRF than `B`.
+- However `C` is **not** preferred over `A`, since they have the same issuer and slot, and therefore we prefer the chain with the highest `opcert` number (2), therefore `A` is preferred over `C`.
```

Comment:
If you have the same issuer and the same slot, then the VRF value must be equivalent.

Reply:
I am not actually sure what exactly you want to demonstrate here, given that you are not using the example from https://github.com/IntersectMBO/ouroboros-consensus/issues/1075#issuecomment-3035911537. It (or variants thereof) are the only source of non-transitivity of `UnrestrictedVRFTiebreaker`.